libr/core/panels.c

```diff
@@ -119,6 +119,79 @@ static const char **menus_sub[] = {
 	NULL
 };
 
+static char *str_crop(const char *str, ut32 x, ut32 y, ut32 x2, ut32 y2) {
```

Comment:
Why not use r_str_crop?

Reply:
no it should be able to use ansi crop which has a bug inside that works properly for VV which i dont know why.

if i fix it the way that should be fixed then VV gets glitches as you once told me